.github/workflows/ci.yml

```
@@ -19,6 +19,16 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
 
+      - name: Cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cargo/registry
+            ~/.cargo/git
+            target
```

The [cargo example](https://github.com/actions/cache/blob/main/examples.md#rust---cargo) has slightly different entries, I am wondering whether we should go with them instead? It is probably the same, as the example only narrows it down to subdirectories:

```suggestion
            ~/.cargo/bin/
            ~/.cargo/registry/index/
            ~/.cargo/registry/cache/
            ~/.cargo/git/db/
            target/
```

---

I didn't find that example. Great find. 

I will align to that example if @rurounijones agrees to keep it simplistic on the dependency chain.

---

Yeah, lets just copy/paste and not add the dependency

---

Sorted and pushed